samcli/local/apigw/local_apigw_service.py

```diff
@@ -142,10 +144,15 @@ def _request_handler(self, **kwargs):
         Response object
         """
         route = self._get_current_route(request)
+        cors_headers = LocalApigwService.cors_to_headers(route.cors)
+
+        if 'OPTIONS' in route.methods:
```

Comment:
is this right? Shouldn't we only be doing this if the request was an Option request instead of if the route has a method of options?

Reply:
Can you answer the above question? This seems wrong when reading but could be just missing something obvious